internal/analyzer/ktn/structure/organization/cargoparams_internal_test.go

```
@@ -1,25 +1,694 @@
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+			pass := &analysis.Pass{
+				Fset:  token.NewFileSet(),
+				Files: []*ast.File{},
+			}
+			inv := make(map[string]*structInfo)
+			collectStructUsages(pass, inv)
+			if len(inv) < 0 {
```

golangci-lint (2.12.1)

[error] SA4024: builtin function len does not return negative values

(staticcheck)